doc/governance.md

```diff
@@ -0,0 +1,228 @@
+period during which they were active in the Council.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed.
+
+The HyperSpy steering council may be contacted at `EMAIL ADDRESS` or via the
+[@hyperspy/steering-council](https://github.com/orgs/hyperspy/teams/steering-council) GitHub team.
+
+## Conflict of interest
+
+It is expected that the SC Members will be employed at a wide range of companies,
+universities and non-profit organizations. Because of this, it is possible that
+Members will have conflict of interests. Such conflict of interests include,
+but are not limited to:
+
+- Financial interests, such as investments, employment, or contracting work,
+  outside of The Project that may influence their work on The Project.
+- Access to proprietary information of their employer that could potentially
+  leak into their work with the Project.
+- An issue where the person privately gains an advantage from The Project resources,
+  but The Project has no gain or suffers a disadvantage.
+  
+All members of the Steering Council shall disclose to the rest of the Council
+any conflict of interest they may have. Members with a conflict of interest in
+a particular issue may participate in Council discussions on that issue,
+but must recuse themselves from voting on the issue. If the BDFL has recused
+themselves for a particular decision, they will appoint a substitute BDFL for that decision.
+
+# Decision Making Process
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the [issue tracker](https://github.com/hyperspy/hyperspy/issues). Occasionally,
+sensitive discussion may occur on a private communication channel.
+
+Decisions should be made in accordance with the [mission and values](MISSION_AND_VALUES.md)
+of the HyperSpy project.
+
+The HyperSpy project uses a “consensus seeking” process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+Core developers are expected to distinguish between fundamental objections to a
+proposal and minor perceived flaws that they can live with, and not hold up the
+decision-making process for the latter.  If no option can be found without
+objections, the decision is escalated to the SC, which will itself use
+consensus seeking to come to a resolution. In the unlikely event that there is
+still a deadlock, the proposal will move forward if it has the support of a
+simple majority of the SC.
+
+Decisions (in addition to adding core developers and SC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction of a
```

Comment:
Not notably relaxed compared with major documentation changes. Except that "reasonable time" might be shorter here.